packages/js/tests/testHelpers/createDomTestData.js

```diff
@@ -172,6 +172,22 @@ const createNonHierarchicalCTElements = () => {
 	return nonHierarchicalParentElement;
 };
 
+/**
+ * Creates an element with id and value attributes.
+ *
+ * @param {string} id       The id name of the element.
+ * @param {string} value    The value name of the element.
+ * @param {string} tagName  The tag name of the element.
+ * @returns {HTMLElement}    The created element with id and value attributes.
+ */
+const createElement = ( id, value, tagName ) => {
```

Comment:
I would consider changing the signature of this function: `tagName` in my opinion should be the first argument, and as a second argument, I would consider having an (optional) object called `attributes` (e.g, `{ id: "newid", value: "test"}`).

Reply:
I think this is a good idea! the code will look cleaner 😉